python/hail/dataset.py

```diff
@@ -5482,5 +5482,52 @@ def to_hail2(self):
         import hail2
         return hail2.VariantDataset(self.hc, self._jvds)
 
+    @handle_py4j
+    @record_method
+    @typecheck_method(pedigree=Pedigree,
+               complete_trios=bool)
+    def trio_matrix(self, pedigree, complete_trios=False):
+        """Builds and returns a matrix with trios are grouped together as cells.
```

Comment:
How about: "Builds and returns a matrix where columns correspond to trios and cells contain genotypes for the trio."

Man, we really shouldn't use "cell" in bio software...   Entry? Item? Record?